vmslave-config/ansible/group_vars/smoke-test

```diff
@@ -0,0 +1,22 @@
+# vagrant box packages
+vagrant_deb: https://releases.hashicorp.com/vagrant/1.9.1/vagrant_1.9.1_x86_64.deb
+virtualbox_deb: http://download.virtualbox.org/virtualbox/4.3.36/virtualbox-4.3_4.3.36-105129~Ubuntu~raring_amd64.deb
+
+# ovftool bundle url
+ovftool_bundle: http://ftp.tucha13.net/pub/software/VMware-ovftool-4.1.0/VMware-ovftool-4.1.0-2459827-lin.x86_64.bundle
+
+# vars for mount isofarm
+isofarm_path: /home/jenkins/isofarm
+isofarm_src: //**.**.**.**/isofarm
+isofarm_fstype: cifs
+isofarm_mount_opts: ro,username=***,password=***
+
+# vars for as gateway
```

Comment:
sorry, your implementation is admirable .
but I'm thinking of how other people to maintain those.
example, I know the question is quite silly for you, but is there any Prerequisite to setup the gateway , how to determine which is EXTIF/INTIF for a new node....
you can't always assume everybody are experts of all technical stack...
so can you please consider about how to make life easier for people who maintain and understand those ?